ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -613,6 +614,15 @@ public void fillFormViaTestID(DataTable data) {
         new Form(new SyndesisRootPage().getRootElement()).fillByTestId(data.asMap(String.class, String.class));
     }
 
+    @Then("^fill in data-testid field \"([^\"]*)\" from property \"([^\"]*)\" of credentials \"([^\"]*)\"")
```

Comment:
@wyvie this should be `@When` probably, but I don't want to block it just because of this, so if you don't forget, please change it in some next PR